Compile error: obsolete API, use ::begin(WiFiClient, url) in "src/utcOffset.cpp"

Arduino IDE:    1.8.15 (Mac OS X)
Ard-ESP-Core:   https://arduino.esp8266.com/stable/package_esp8266com_index.json
ESP8266 Board:  LOLIN(WEMOS) D1 R2 & mini

Compile Error in "src/utcOffset.cpp":
```
/WordClock401_ESP8266/src/utcOffset.cpp: In static member function 'static void UtcOffset::updateLocalizedUtcOffset()':
/WordClock401_ESP8266/src/utcOffset.cpp:9:13: error: call to 'HTTPClient::begin' declared with attribute error: obsolete API, use ::begin(WiFiClient, url)
    9 |   http.begin("http://worldtimeapi.org/api/ip");

exit status 1
Fehler beim Kompilieren für das Board LOLIN(WEMOS) D1 R2 & mini.
```

--- a/wordclock/src/utcOffset.cpp
+++ b/wordclock/src/utcOffset.cpp
@@ -5,8 +5,15 @@
 #include "config.h"
 
 void UtcOffset::updateLocalizedUtcOffset() {
+/* remove the depricated ESP8266 API call:  
+ *HTTPClient http;
+ *http.begin("http://worldtimeapi.org/api/ip");
+*/
+
+//  use the new API like below:
+  WiFiClient client;
   HTTPClient http;
-  http.begin("http://worldtimeapi.org/api/ip");
+  http.begin(client, "http://worldtimeapi.org/api/ip");
   int responseCode = http.GET();
 
   if (responseCode == 200) {

